### DISCUSSION TITLE
F10 to disable mouse buttons doesn't also disable scroll wheel

### DISCUSSION THREAD

**Discussion by tylort69** [ppy/osu#15917] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/28652424/144562120-0e0a9f4c-2799-4a1e-bd8d-0d7c73a88004.png)
I'm aware of this option[Disable Mouse Scroll Wheel During Gameplay] in the menu but this doesn't give the functionality I want either.
in stable pressing F10 will disable mouse buttons and scroll wheel during gameplay but if you use alt+ scroll wheel it will still work to change volume which is the functionality I desire. to disable scroll wheel during gameplay but allow alt+ scroll wheel to work like regular.
the reason for this is I have scroll wheel bound to a button on my pen and it gets pressed accidently all the time but its very useful to use the alt modifier to quickly change the volume and adding the need for the modifier makes it so that the chance of accidental usage is practically 0.

**Comment by peppy** [comment, marked as answer]
I think the best course of action here is to remove the "disable mouse wheel during gameplay", and potentially the buttons one too. We have better handling for these cases these days, via the key binding setup. I'll take a look into this.

**Reply by peppy** [reply]
Removing the settings seems like it will throw people off, even though it's likely the saner way to handle it from a code perspective, so I'm going to leave them for now. I've renamed the volume adjust one to be more specific to what it's doing.

Addressed via #15918.
